packages/x-date-pickers-pro/package.json

```diff
@@ -42,11 +42,11 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.17.2",
-    "@date-io/date-fns": "^2.14.0",
-    "@date-io/dayjs": "^2.14.0",
-    "@date-io/luxon": "^2.14.0",
-    "@date-io/moment": "^2.14.0",
-    "@mui/utils": "^5.7.0",
+    "@date-io/date-fns": "^2.11.0",
```

Comment:
Please don't downgrade `@date-io`, I'm doing releases on it so I need the latest features at any time

Reply:
Right, OK it matches with this part of the PR description:

> For @date-io/* it's less obvious to me if my proposed change is a step forward or backward. The big difference is that developers won't install it on their own, so there are no duplication upsides, and the downside is that for fixes done in @date-io/ it's harder for developers to upgrade the access to them. So, maybe to revert?

Reply:
More than fixes, I need the new features, the pickers will not work at all without 2.14
And I suspect it will be the same for the next upgrades of `@date-io`

Reply:
Yeah OK, agree, let's do this. I'm updating the PR